### PR TITLE
Added provided assignments as on 14jan

### PR DESCRIPTION
**Pull request by senseihimanshu**

### CODE REVIEW

File: Himanshu_Sharma/form_using_bootstrap/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="./assets/styles/main.css">
+</head>
+<body>
+    <div style="height: 100vh;" class="d-flex flex-column justify-content-center bg-dark">
```
Comment (nikhilrstg18) [inline Himanshu_Sharma/form_using_bootstrap/index.html:12]: get rid of all style attributes in html files

File: Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js [hunk 2]
```diff
@@ -0,0 +1,229 @@
+//This obj stores the state of any block 
+var blocks = {};
+
+//A simple boolean to check turn
+var turn = false;
+
+(function blocksCreation(){
+    for(let row = 0; row < 3; row++){
```
Comment (nikhilrstg18) [inline Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js]: time complexity is 'n^2', can you improve it to 'n'

File: Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    var player2 = 'O';
+
+    document.querySelector(`.${clickedClass}`).textContent = ((turn === false)? player1 : player2);
+    blocks[`${clickedClass}`] = ((turn === false)? 1 : -1);
+    turn = !turn;
+}
+
+checkForNoResult = () => {
+    let countZeroes = 0;
+    for (const property in blocks) {
```
Comment (nikhilrstg18) [inline Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js]: using for in causing performance degrade, think of work-around for it

File: Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+        if(blocks[property] === 0){
+            countZeroes++;
+        }
+    }
+    if(countZeroes === 0){
+        this.printWinner();
+    }
+}
+
+// removeListeners = () => {
```
Comment (nikhilrstg18) [inline Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js]: 1. remove commented code in html, css and JS files
2. add functional or step comments in htl css and JS code

File: Himanshu_Sharma/form_using_bootstrap/index.html (hunk 1 above)
Comment (senseihimanshu) [reply Himanshu_Sharma/form_using_bootstrap/index.html:12]: This was just an assignment for bootstrap it is not for review, I will take care of it next time, but won't bootstrap implemented like it!

File: Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js (hunk 3 above)
Comment (senseihimanshu) [reply Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js]: Fixed it!

File: Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js (hunk 4 above)
Comment (senseihimanshu) [reply Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js]: Fixed it!

File: Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js (hunk 2 above)
Comment (senseihimanshu) [reply Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js]: It cannot possibly, for current built project as it is necessary to add a unique class to each block of the game! I will take care of it later...

File: Himanshu_Sharma/form_using_bootstrap/index.html (hunk 1 above)
Comment (nikhilrstg18) [reply Himanshu_Sharma/form_using_bootstrap/index.html:12]: suggestion: either use `class` or `style` not both at the same time
you are violating KISS principle + breaking natural cascade from external css

File: Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+        }
+    }
+
+    if(countZeroes === 0){
+        this.printWinner();
+    }
+}
+
+findWinner = () => {
+    var styleStringForStrikeThrough = "background-color: black; position: absolute; transition: all .2s";
```
Comment (nikhilrstg18) [inline Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js]: @senseihimanshu 
violating DRY
work on getting rid of repeated steps in the code, FindWinner can be reduced to a great extent

**Review by nikhilrstg18 — CHANGES_REQUESTED** [review]
scope of improvement added, please address all PR comments

File: Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js (hunk 5 above)
Comment (senseihimanshu) [reply Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js]: @nikhilrstg18 I had reviewed the code and fixed accordingly Thankyou!

**Review by nikhilrstg18 — APPROVED** [review]
Looks good to me

**Review by amandeeprikhi — APPROVED** [review]
LGTM

**Pull request merged**